Marketplace/Support/manage-marketplace-subscriptions.md

```diff
@@ -0,0 +1,94 @@
+Fill out the **Request Description** field with your request. The request description should state which action you are looking to request.
+
+The options are:
+
+a. Update Billable Items
+b. Terminate Billable Items
+c. Add Billable Items *Additional charges may apply to certain selections*
+
+Ex. I am requesting an Update Billable Items to downgrade from 100 to 80 license seats.
+Ex. I am requesting to Terminate Billable Items.
+Ex. I am requesting to Add Billable Items to my subscription. Please add 10 license seats.
+
+3. Lastly, you will check the billable item(s) that you are wanting to update from the dropdown list that matches the update option that you are requesting.
+
+  ![Marketplace Manager4](../images/MM4.png)
+
+4. Click **Request Update** button to submit the request. You will receive a conformation email that your request has been received. *Please note: Submitting an Update Request is a manual process, it does not automatically alter functionality or billing of the product.*
+
+5. The Provider of the Service that you requested an update for will contact you via email to communicate that your request has been fulfilled and the date that the new functionality and updated billing will begin.
+
+###Terminate
+The **Terminate** button allows customers to terminate their Marketplace subscriptions that do NOT have an active term commit associated with them. If there is an active term commit associated with the subscription the terminate button will not populate for that subscription until 30 days prior to the end of the term commit date. For subscriptions with term commits, you will be notified via email 30 days prior to your term commit end date. If you chose to keep the service no action is needed on your part and the subscription will auto-renew for the same terms. If you wish to terminate the subscription you must do so via the Terminate request prior to the term commit end date.
+
+####Steps to Make a Terminate Request
+
+1. To make a termination request for a subscription click on the **Terminate** button. The form below will populate.
+
+  ![Marketplace Manager5](../images/MM5.png)
+
+2. Next, fill out the form. The **Contact Email** will auto-populate, however, you can change it to another email if you chose.
+Fill out the **Request Description** field with your request.
+
+Ex. I am requesting termination of my subscription.
+
+3. Click the **Terminate** button to submit the request. You will receive a conformation email that your request has been received. *Please note: Submitting a terminate request is a manual process, it does not automatically terminate the subscription or immediately stop billing of the product.*
+
+5. The Provider of the Subscription that you requested a termination for will contact you via email to communicate that your termination request has been fulfilled and the date that billing will end.
```

Comment:
The Provider of the Subscription for which you requested a termination will contact you via email.  They will communicate to you when your termination request has been fulfilled with the date that billing will end.